nl/guides/for-students/courses/README.md

```diff
@@ -0,0 +1,155 @@
+
+![image](./student.deadline_series_warning.png)
+
+Als je **na het verstrijken van de deadline** van een [oefeningenreeks]() een [oplossing](../exercises/#navigeren-naar-een-oplossing) [indient](../exercises/#navigeren-naar-een-oefening) voor een [oefening](../exercises/#navigeren-naar-een-oefening) uit een [oefeningenreeks](), dan zal je [indienstatus](#indienstatus) voor de oefening in de oefeningenreeks daardoor nooit wijzigen. Je indienstatus voor een oefening in een oefeningenreeks wordt immers bepaald op basis van je laatst ingediende oplossing vóór de [deadline](#deadline).
+:::
+
+
+In het menu van een [oefeningenreeks]() vind je de volgende opties:
+
+`Toon overzicht`
+
+Toont een overzicht waarin de namen en beschrijvingen van alle [oefeningen](../exercises/#navigeren-naar-een-oefening) uit de [oefeningenreeks]() netjes onder elkaar staan. Onder elke beschrijving staat ook je [indienstatus](#indienstatus) voor de oefening. Als je op de indienstatus drukt dan navigeer je naar de [oplossing](../exercises/#navigeren-naar-een-oplossing) die gebruikt werd om de indienstatus te bepalen (als je effectief een oplossing hebt [ingediend](../exercises/#navigeren-naar-een-oplossing) op basis waarvan de indienstatus kon bepaald worden).
+
+::: tip
+Dit overzicht is handig als je een afgedrukte versie wil van alle [oefeningen](../exercises/#navigeren-naar-een-oefening) uit een [oefeningenreeks](). Dodona voorziet dezelfde verzorgde opmaak als bij het [afdrukken]() van een individuele oefening.
+:::
+
+
+`Oplossing downloaden`
+
+Downloadt een ZIP-bestand dat voor elke [oefening](../exercises/#navigeren-naar-een-oefening) uit de [oefeningenreeks]() de [oplossing](../exercises/#navigeren-naar-een-oplossing) bevat die gebruikt werd om je [indienstatus](#indienstatus) voor de oefening te bepalen (als je effectief een oplossing hebt [ingediend](../exercises/#navigeren-naar-een-oplossing) op basis waarvan de indienstatus kon bepaald worden).
+
+## Registreren voor een cursus
+
+Je moet geregistreerd zijn voor een [cursus]() voordat je [oplossingen](../exercises/#navigeren-naar-een-oplossing) kan [indienen](../exercises/#indienen-van-een-oplossing) voor de [oefeningen](../exercises/#navigeren-naar-een-oefening) uit de cursus.
+
+<h1 id="manuele-registratie"/>
+
+Als je [navigeert](#navigeren-naar-een-cursus) naar een [cursus]() waarvoor je nog niet [geregistreerd](#registreren-voor-een-cursus) bent, dan zie je bovenaan de [cursuspagina](#cursuspagina) een paneel dat aangeeft of en hoe je je voor de cursus kan [registreren](#registreren-voor-een-cursus). Hierbij zijn er drie mogelijkheden:
+
+-   De [cursus]() werkt met een [open]() [registratieprocedure](), wat betekent dat iedereen voor de cursus kan registreren zonder expliciete goedkeuring van een cursusbeheerder. Druk op de knop `REGISTREREN` om je voor de cursus te registreren.
+
+    ![image](./register.png)
```

Comment:
some of these screenshots don't look like a screenshot because they only contain text

Reply:
A possible solution for this could be to put the images in a quote that would look like this:
 
![image](https://user-images.githubusercontent.com/48768719/91869424-2a685c80-ec76-11ea-89d7-3ca3139e1a92.png)

Alternatively, I could try to make some changes to the screen-shot bot to take a screenshot of the parent of the callout which should look like this:
![image](https://user-images.githubusercontent.com/48768719/91869557-52f05680-ec76-11ea-94ee-52fbce4521bc.png)


Here is a screenshot of how it currently looks:
![image](https://user-images.githubusercontent.com/48768719/91869501-42d87700-ec76-11ea-837b-23cbad0a9b47.png)